Projects/Server/GameCounter.cpp

```diff
@@ -0,0 +1,96 @@
+#include "GameCounter.h"
+
+GameCounter::GameCounter(const AbstractPlayer::Ptr& player0, const AbstractPlayer::Ptr& player1, const size_t& gameSample = 1)
+	: _player0{ player0 }, _player1{ player1 }
+{
+	gameStats["p0win"];
+	gameStats["p1win"];
+	gameStats["draw"];
+	gameStats["gamesPlayed"];
+	gameStats["gameSample"] = gameSample;
+}
+
+
+void GameCounter::play() {
+
+
+	GameHandler gameHandler{};
+	size_t gameSample = gameStats["gameSample"];
+	AbstractPlayer::Ptr player0 = getPlayer0();
+	AbstractPlayer::Ptr player1 = getPlayer1();
```

Comment:
No need for this, you can simply use `_player0` and `_player1` throughout.

Reply:
Removed redundant variables.